src/scribe-plugin-drag-and-drop.js

```diff
@@ -82,6 +82,20 @@ module.exports = function(config) {
     });
 
 
+    helpers.delegate(scribe.el, "dragover", "p", function (event) {
+      if(event.target.className.indexOf(config.style_class) !== -1
+         && event.target.className.indexOf(config.hover_class) === -1) {
```

Comment:
Cool. Did't even know it existed!

On 16 June 2015 at 12:38, Sébastien Cevey <notifications@github.com> wrote:

> In src/scribe-plugin-drag-and-drop.js
> <https://github.com/guardian/scribe-plugin-drag-and-drop/pull/8#discussion_r32512443>
> :
>
> > @@ -82,6 +82,20 @@ module.exports = function(config) {
> >      });
> >
> >
> > +    helpers.delegate(scribe.el, "dragover", "p", function (event) {
> > +      if(event.target.className.indexOf(config.style_class) !== -1
> > +         && event.target.className.indexOf(config.hover_class) === -1) {
>
> Why not use classList.has for this (assuming we don't care for older
> browser that don't have it, and I don't think we do)? It would be a lot
> less brittle than substring matching, which doesn't correctly identify
> classes (e.g. foo is in foobar baz).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/guardian/scribe-plugin-drag-and-drop/pull/8/files#r32512443>
> .
>

-- 

------------------------------
Visit theguardian.com. On your mobile and tablet, download the Guardian 
iPhone and Android apps theguardian.com/guardianapp and our tablet editions 
theguardian.com/editions.  Save up to 57% by subscribing to the Guardian 
and Observer - choose the papers you want and get full digital access. 
 Visit subscribe.theguardian.com

This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396